dataactvalidator/filestreaming/loaderUtils.py

```diff
@@ -24,7 +24,7 @@ def compareRecords (recordA,recordB, fields) :
         return True
 
     @classmethod
-    def loadCsv(cls,filename,model,interface,fieldMap,fieldOptions):
+    def loadCsv(cls,filename,model,interface,fieldMap,fieldOptions,skipClear = False):
```

Comment:
If `skipClear` is here just for the multiple files, then +1 to the earlier comment about pre-combining them and not building special logic into the file load process.